server/lib/src/idm/account.rs

```diff
@@ -542,6 +545,7 @@ impl Account {
 
         // Anonymous does NOT record it's sessions, so we simply check the expiry time
         // of the token. This is already done for us as noted above.
+        dbg!(uat.clone());
```

Comment:
debug/trace rather than dbg in production code here.

Reply:
I was just fighting against failing tests, I didn't mean to leave it in prod :p

Reply:
Naughty naughty :P  It's why trace! is good here since that's meants to be developer-level logs we can accidentally leave in. We plan to do a big overhaul of log messages soon to make sure they are all at proper levels and details.